CONTRIBUTING.md

```diff
@@ -0,0 +1,97 @@
+- What you expected would happen
+- What actually happens
+
+**Notes:** possibly include theories to explain the behaviour, or steps you tried that didn't work or any other information you think that’s valuable that will help our team to provide a fix soon
+
+People ***love*** thorough bug reports. [This is a good example](https://pagure.io/dogtagpki/issue/3194) of a thorough bug report
+
+## Documentation
+
+We have 3 different documentation categories:
+
+1. [User/Admin guides](docs/), which describe how to use the features
+2. [Man pages](docs/manuals/), which include the manuals for configuration files and CLI commands
+3. [Wiki pages](https://www.dogtagpki.org/wiki/PKI_Main_Page), which include the design, technical and non-product specific information
+
+Wiki pages are the best place to start learning about the product. If you have any issues or find any bugs in our documentation please feel free to open a PR or [contact us](#Contact)
+
+## Style Guidelines
+
+### Git Commit Messages
+
+- Use the present tense ("Add feature" not "Added feature")
+- Use the imperative mood ("Move cursor to..." not "Moves cursor to...")
+- Limit the subject to less than 50 characters
+- Separate subject from body with a blank line
+- Reference issues and pull requests liberally after the first line
+- Wrap the commit message body at 72 characters
+- Use the body to explain what and why vs. how
+
+### Pull Request Description
+
+While issuing a Pull Request, Github by default, autofills the PR description from all your commit messages. Though this suffices most times, you may have to manually clean it.
+
+- Ensure the description contains a detailed description
+- Include test procedure, if applicable
+- Provide a link to the upstream issue, if any
+- Provide additional information, which you think reviewers must be aware of before reviewing your PR
+
+[This is a good example](https://github.com/dogtagpki/pki/pull/471) of issuing a PR.
+
+### Python Coding Style
+
+We lint our Python code against both *PyLint* and *Flake8*; we run PyLint against our own [pylintrc](tools/pylintrc) file.
+
+## Contact
+
+You can reach the Dogtag PKI team over the **#dogtag-pki** channel on freenode.net. Note that you need to be a [registered user](https://freenode.net/kb/answer/registration) to message on this channel. You can also send an email to pki-users@redhat.com
+
+See also [how we work](https://www.dogtagpki.org/wiki/PKI_Team_Dynamics)
+
+## License
+
+The source code of this project is licensed under various open source licenses, with the predominant license being the GNU General Public License version 2 (GPLv2). When you submit  changes to existing source files, you license your submissions  under the license of the file you are modifying. Entirely new source files are licensed under GPLv2 or any later version of the GPL. Feel free to contact us if you have any questions about licensing of your contributions.
```

Comment:
If possible add a reference to this page in this section:
https://github.com/dogtagpki/pki/blob/master/LICENSE
and avoid/minimize mentioning specific license names in this section so in case the license is updated this page won't become obsolete.

Reply:
Done